src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -264,8 +264,7 @@ private SNITCPHandle CreateTcpHandle(string fullServerName, long timerExpire, ob
 
             if (serverNamePartsByComma.Length < 2 && serverNamePartsByBackSlash.Length < 2)
             {
-                int defaultInstancePort = TryToGetDefaultInstancePort(fullServerName);
-                return new SNITCPHandle(fullServerName, (defaultInstancePort > 0 ? defaultInstancePort : DefaultSqlServerPort), timerExpire, callbackObject, parallel);
+                return new SNITCPHandle(fullServerName, DefaultSqlServerPort, timerExpire, callbackObject, parallel);
```

Comment:
The other place where `TryToGetDefaultInstancePort` is being used is where there is no server name specified in DataSource for the SqlConnection. 

In that case, the DataSource should be assumed as `localhost`, however the Default port should be used, and no port should be queried from the browser process. 

I think we can remove the method and its calls.